vocab/credentials/v2/credentials.ttl

```diff
@@ -0,0 +1,120 @@
+@prefix dc: <http://purl.org/dc/terms/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix cred: <https://w3.org/2018/credentials#> .
```

Comment:
how come we don't see

```
https://w3id.org/security#
```

In this list....

Reply:
Because the vocabulary does not refer/use this namespace.

Reply:
(see also my comment in https://github.com/w3c/vc-data-model/pull/920#discussion_r960290781)